setup.py

```diff
@@ -36,8 +36,9 @@
     python_requires='>=3.6, <3.7',
     install_requires=[
         'Django~=1.11.0',
-        'djangorestframework~=3.7.0',
-        'djangorestframework-filters~=0.10.0',
+        'djangorestframework~=3.9.0',
+        'djangorestframework-filters~=1.0.0.dev0',
```

Comment:
We cannot depend on `dev` version. Are there any plans for a next release?

Reply:
There were always conflicts between drf, drf-filters and django_filters, but there was no other way to achieve what we needed in the past. It would be great if we can remove drf-filters. Even if it requires few days of work.

Reply:
DRF-filters removed, tests pass.